pkg/reconciler/buildrun/resources/remote_artifacts_test.go

```diff
@@ -0,0 +1,111 @@
+package resources
+
+import (
+	"fmt"
+	"reflect"
+	"testing"
+
+	buildv1alpha1 "github.com/shipwright-io/build/pkg/apis/build/v1alpha1"
+	"github.com/shipwright-io/build/pkg/config"
+	"github.com/tektoncd/pipeline/pkg/apis/pipeline/v1beta1"
+)
+
+func TestRemoteArtifacts_outputPathForBuildSource(t *testing.T) {
```

Comment:
We use Ginkgo/Gomega style pretty much everywhere else. Wondering if we should aim for a more common style.

Reply:
I end up writing vanilla go test since it was simpler to implement the test tables for unit testing.